lstchain/reco/r0_to_dl1.py

```diff
@@ -453,192 +448,19 @@ def r0_to_dl1(
-                                )
-
-                            if event.lst.tel[telescope_id].evt.extdevices_presence & 1 \
-                                    and first_valid_ucts_tib is None:
-                                # Both TIB and UCTS presence flags are OK
-                                first_valid_ucts_tib = ucts_time
-
-                                initial_tib_counter = (
-                                        event.lst.tel[telescope_id].evt.tib_pps_counter +
-                                        event.lst.tel[telescope_id].evt.tib_tenMHz_counter * 10 ** (-7)
-                                )
-                                logger.warning(
-                                    f"TIB timestamps not based on a valid absolute reference timestamp. "
-                                    f"Consider using the following initial values \n"
-                                    f"Event ID: {event.index.event_id}, UCTS timestamp corresponding to "
-                                    f"the first valid TIB counter: {first_valid_ucts_tib:.9f} s, "
-                                    f"corresponding TIB counter {initial_tib_counter:.9f} s"
-                                )
-                        else:
-                            ucts_time = math.nan
-
-                    else:
-                        # Dragon/TIB timestamps based on a valid absolute reference UCTS timestamp
-                        dragon_time = (
-                                (ucts_t0_dragon - dragon_counter0) * 1e-9 +  # secs
-                                event.lst.tel[telescope_id].evt.pps_counter[module_id] +
-                                event.lst.tel[telescope_id].evt.tenMHz_counter[module_id] * 10 ** (-7)
-                        )
-
-                        tib_time = (
-                                (ucts_t0_tib - tib_counter0) * 1e-9 +  # secs
-                                event.lst.tel[telescope_id].evt.tib_pps_counter +
-                                event.lst.tel[telescope_id].evt.tib_tenMHz_counter * 10 ** (-7)
-                        )
-
-                        if event.lst.tel[telescope_id].evt.extdevices_presence & 2:
-                            # UCTS presence flag is OK
-                            ucts_time = event.lst.tel[telescope_id].evt.ucts_timestamp * 1e-9  # secs
-                            if first_valid_ucts is None:
-                                first_valid_ucts = ucts_time
-                            if first_valid_ucts_tib is None \
-                                    and event.lst.tel[telescope_id].evt.extdevices_presence & 1:
-                                first_valid_ucts_tib = ucts_time
-                        else:
-                            ucts_time = math.nan
+                    # !!! to be filled  from the source !!!
+                    dl1_container.ucts_time = 0
+                    dl1_container.dragon_time = 0
+                    dl1_container.tib_time = 0
```

Comment:
I would keep only dragon_time which is the most reliable one now, and the one calculated by the event source. In any case, once we move to using the standard DL1 format, only one timestamp will be kept. 
The cross-checks involving comparisons of different times etc should be made if needed at an earlier stage (and I think they are already done in UCM-GAE's daily check)

Reply:
I did some comparisons of the r0_to_dl1 output (the one in this PR vs. the one in the latest lstchain release) and it seems to me that we are using the "waveform-shifting" option of the new LST R1 event source. I would rather switch that off for now, since it will introduce some differences in reconstructed pixel charges and times,  which will make it harder to detect other possible differences between the new and the old DL1.

Reply:
For that you should just add `"apply_waveform_time_shift": false` to the `CameraCalibrator` (default?) config.

Reply:
Reminder, this is the config I use for the stage1 calibration: 

```
  "LSTEventSource": {
    "LSTR0Corrections": {
      "drs4_pedestal_path": "../data/lst/2020-09-09/drs4_pedestal.Run2603.0000.fits",
      "drs4_time_calibration_path": "../data/lst/2020-09-09/time_calibration.Run2605.0000.hdf5",
      "calibration_path": "../data/lst/2020-09-09/calibration.Run2605.0000.hdf5",
      "gain_selection_threshold": 3500,
      "add_calibration_timeshift": true
    },
    "PointingSource": {
      "drive_report_path": "../data/lst/2020-09-09/drive_log_20_09_09.txt"
    },
    "EventTimeCalculator": {
      "ucts_t0_dragon": 1599691710450937890,
      "dragon_counter0": 7450937600,
      "ucts_t0_tib": 1599691710450937890,
      "tib_counter0": 7450940400
    }
  },
  "CameraCalibrator": {
    "apply_waveform_time_shift": false,
    "image_extractor_type": "LocalPeakWindowSum",
    "LocalPeakWindowSum": {
      "window_shift": 4,
      "window_width": 8,
      "apply_integration_correction": false
    }
  },

```